plugins/Feedback/angularjs/refer-banner/refer-banner.controller.js

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * @link http://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+(function () {
+    angular.module('piwikApp').controller('ReferBannerController', ReferBannerController);
+
+    ReferBannerController.$inject = ['$scope'];
+
+    function ReferBannerController($scope) {
+        var setNextReminder = function(nextReminder) {
+            var ajaxHandler = new ajaxHelper();
+            ajaxHandler.addParams({'module': 'Feedback', 'action': 'updateReferReminderDate'}, 'GET');
+            ajaxHandler.addParams({'nextReminder': nextReminder}, 'POST');
+            ajaxHandler.send();
+        };
+
+        var closeBanner = function() {
+            $scope.referBanner.show = false;
+
+            setNextReminder(6 * 30);
+        };
+
+        var share = function() {
+            $scope.referBanner.show = false;
+            $scope.referBanner.showThanks = true;
+
+            setNextReminder(-1);
+        }
+
+        $scope.socialUrl = function (type) {
+            var text = _pk_translate('Feedback_ReferBannerSocialShareText');
+            var url = 'https://matomo.org/google-owns-your-data/?pk_campaign=share&pk_kwd=onpremise';
+
+            if (type === 'twitter') {
+                var base = 'https://twitter.com/intent/tweet?';
+
+                var params = { 'text': text, 'url': url};
+                var paramString = new URLSearchParams(params);
+
+                return base + paramString.toString();
+            }
+
+            if (type === 'facebook') {
+                var base = 'https://www.facebook.com/sharer.php?';
+
+                var params = { 't': text, 'u': url};
+                var paramString = new URLSearchParams(params);
```

Comment:
`URLSearchParams` has no support in IE: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams and currently we still support IE11. Can you do this another way?